### PR TITLE
[K8s] Software Update: ELK v8.7.0

### PR DESCRIPTION
**Pull request by karun-singh**
- Updated elk stack to v8.7.0

### CODE REVIEW

File: K8s-deployment/Charts/elk/elasticsearch/es-values.yml [hunk 1]
```diff
@@ -1784,6 +1808,12 @@ metrics:
     ##   - myRegistryKeySecretName
     ##
     pullSecrets: []
+  ## @param metrics.annotations [object] Annotations for metrics
+  ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+  ##
+  annotations:
+    helm.sh/hook: "post-install,post-upgrade"
```
Comment (abhi4578) [inline K8s-deployment/Charts/elk/elasticsearch/es-values.yml:1815]: what is this doing? what is this hook for?

**Comment by abhi4578** [comment]
also the whole directory v7.17 can be removed in master branch.

File: K8s-deployment/Charts/elk/elasticsearch/es-values.yml (hunk 1 above)
Comment (karun-singh) [reply K8s-deployment/Charts/elk/elasticsearch/es-values.yml:1815]: These hooks signify that the metrics pod would be brought up only after the elasticsearch pods are created and upgraded only after elasticsearch pods are upgraded. [Refer](https://helm.sh/docs/topics/charts_hooks/#the-available-hooks).

Comment (abhi4578) [reply K8s-deployment/Charts/elk/elasticsearch/es-values.yml:1815]: okay. got it.

**Review by abhi4578 — APPROVED** [review]

**Pull request merged**